packages/input-option/src/InputOption/InputOption.types.ts

```diff
@@ -42,6 +77,17 @@ export interface BaseInputOptionProps {
    * @default true
    */
   isInteractive?: boolean;
+
+  /**
+   * Styles input based on intended action
+   * @default 'default'
+   */
+  actionType?: ActionType;
```

Comment:
If I understand correctly, since a form can't be destructive, can we make this conditional so that a type error shows when `renderedContext` is `form` and `actionType` is `destructive`? Similar to what we do in Popover with `usePortal`?